addons/sale/models/sale_order.py

```diff
@@ -879,21 +879,21 @@ def _find_mail_template(self, force_confirmation_template=False):
             if default_confirmation_template:
                 return default_confirmation_template
             else:
-                return self.env.ref('sale.mail_template_sale_confirmation')
-        return self.env.ref('sale.email_template_edi_sale')
+                return self.env.ref('sale.mail_template_sale_confirmation', None)
+        return self.env.ref('sale.email_template_edi_sale', None)
```

Comment:
`raise_if_not_found=None` ?

Reply:
I have obviously mistaken this parameter. I thought that it returned the mail.template or None. I'll change that right away.